### ISSUE TITLE
create_cookies error in You provider

### ISSUE THREAD

**Issue by sjrsjz** [xtekky/gpt4free#1739]
log:
Using You provider and claude-3-sonnet model
Traceback (most recent call last):
  File "E:\mirai\AliceBot\AliceBot.py", line 56, in <module>
    test=g4f.ChatCompletion().create(
  File "f:\Python_3.10.9\lib\site-packages\g4f\__init__.py", line 149, in create
    return result if stream else ''.join([str(chunk) for chunk in result])
  File "f:\Python_3.10.9\lib\site-packages\g4f\__init__.py", line 149, in <listcomp>
    return result if stream else ''.join([str(chunk) for chunk in result])
  File "f:\Python_3.10.9\lib\site-packages\g4f\providers\base_provider.py", line 206, in create_completion
    yield loop.run_until_complete(await_callback(gen.__anext__))
  File "f:\Python_3.10.9\lib\site-packages\nest_asyncio.py", line 98, in run_until_complete
    return f.result()
  File "f:\Python_3.10.9\lib\asyncio\futures.py", line 201, in result
    raise self._exception.with_traceback(self._exception_tb)
  File "f:\Python_3.10.9\lib\asyncio\tasks.py", line 234, in __step
    result = coro.throw(exc)
  File "f:\Python_3.10.9\lib\site-packages\g4f\providers\base_provider.py", line 202, in await_callback
    return await callback()
  File "f:\Python_3.10.9\lib\site-packages\g4f\Provider\You.py", line 69, in create_async_generator
    cookies = await cls.get_cookies(session) if chat_mode != "default" else None
  File "f:\Python_3.10.9\lib\site-packages\g4f\Provider\You.py", line 145, in get_cookies
    cls._cookies = await cls.create_cookies(client)
  File "f:\Python_3.10.9\lib\site-packages\g4f\Provider\You.py", line 185, in create_cookies
    await raise_for_status(response)
  File "f:\Python_3.10.9\lib\site-packages\g4f\requests\raise_for_status.py", line 23, in raise_for_status_async
    raise ResponseStatusError(f"Response {response.status}: {message}")
g4f.errors.ResponseStatusError: Response 401: DFP telemetry ID not set

However when use gpt-3.5 it won't throw error

**Comment by TheColonists** [comment]
I am also having the 401 error.
```
Using RetryProvider provider and gpt-4 model
Using Raycast provider
Using ChatgptAi provider
ChatgptAi: RateLimitError: Rate limit reached
Using You provider
You: ResponseStatusError: Response 401: DFP telemetry ID not set
Using Bing provider
```

**Comment by TheColonists** [comment]
Still active issue

**Comment by AksLolCoding** [comment]
FIX: Use the `"gpt-3.5-turbo"` model and it should work.
The issue is that the `create_cookies` function doesn't work. However, if you use the `"gpt-3.5-turbo"` model, this function is not called and the bot works as exprected.
